nuxt.config.ts

```diff
@@ -25,6 +25,8 @@ export default defineNuxtConfig({
     productSlug: 'clicker-demo',
     clientId: process.env.PLANSHIP_CLIENT_ID,
     clientSecret: process.env.PLANSHIP_CLIENT_SECRET,
+    baseUrl: process.env.PLANSHIP_BASE_URL || '',
```

Comment:
We should get in the habit of using the nullish coalescing operator (`??`) for these types of situations.